File: src/0977-Square-of-Sorted-Array/squares_of_sorted_array.cpp
#include <iostream>
#include <vector>
#include <algorithm>

class Solution {
public:
    std::vector<int> sortedSquares(std::vector<int>& nums) {
        
        for(int i = 0; i< nums.size(); i++){
            nums[i] *= nums[i];
        }
        std::sort(nums.begin(), nums.end());
        return nums;
    }
};

int main() {
    std::vector<int> nums = {-4,-1,0,3,10};
    Solution solution;

    std::vector<int> result = solution.sortedSquares(nums);

    for (int i : result) {
        std::cout << i << " ";
    }

    return 0;
}

File: src/0977-Square-of-Sorted-Array/squares_of_sorted_array.py
from typing import List

class Solution:
    def sortedSquares(self, nums: List[int]) -> List[int]:
        for i in range(len(nums)):
            nums[i] *= nums[i]
        nums.sort()
        return nums
 
if __name__ == "__main__":
    nums: List[int] = [-4,-1,0,3,10]
    solution = Solution()

    print(solution.sortedSquares(nums))